### PR TITLE
NCC: 17 April

### LINKED ISSUE #372: Killer Service

### LINKED ISSUE #376: Life Insurance

### LINKED ISSUE #381: Maestros Confluence

### LINKED ISSUE #383: Make an Example

### LINKED ISSUE #384: Mask of the Schemer

### LINKED ISSUE #385: Master of Ceremonies

### LINKED ISSUE #386: Mezzio Mugger

### LINKED ISSUE #387: Misfortune Teller

### PR DESCRIPTION
**Pull request by Northmoc**
more with no engine tweaks

closes #376 
closes #372 
closes #381
closes #383
closes #384
closes #385
closes #386 
closes #387

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/misfortune_teller.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Misfortune Teller
+ManaCost:3 B
+Types:Creature Human Warlock
+PT:3/1
+K:Deathtouch
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME enters the battlefield or deals combat damage to a player, exile target card from a graveyard. If it was a creature card, create a 2/2 black Rogue creature token. If it was a land card, create a Treasure token. Otherwise, you gain 3 life.
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigExile | TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or deals combat damage to a player, target card from a graveyard. If it was a creature card, create a 2/2 black Rogue creature token. If it was a land card, create a Treasure token. Otherwise, you gain 3 life.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/misfortune_teller.txt:7]: missing "exile" in TriggerDescription

File: forge-gui/res/cardsfolder/upcoming/mezzio_mugger.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Mezzio Mugger
+ManaCost:4 R
+Types:Creature Viashino Rogue
+PT:3/3
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, exile the top card of each player's library. You may play those cards this turn, and you may spend mana as though it were mana of any color to cast those spells.
+SVar:TrigExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | Defined$ Player | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ EffPlay | EffectOwner$ You | RememberObjects$ Remembered | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
+SVar:EffPlay:Mode$ Continuous | MayPlay$ True | MayPlayIgnoreType$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play those cards this turn, and you may spend mana as though it were mana of any color to cast those spells.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mezzio_mugger.txt:8]: MayPlayIgnoreColor$

rest looks solid :)

**Comment by Hanmac** [comment]
imo Master of Ceremonies should probably be cleaned up later

**Comment by Northmoc** [comment]
> imo Master of Ceremonies should probably be cleaned up later

I agree, it's gross

**Review by tool4ever — DISMISSED** [review]

**Review by tool4ever — APPROVED** [review]

**Pull request merged**